app/services/invoices/payments/cashfree_service.rb

```diff
@@ -0,0 +1,216 @@
+# frozen_string_literal: true
+
+module Invoices
+  module Payments
+    class CashfreeService < BaseService
+      include Customers::PaymentProviderFinder
+
+      PENDING_STATUSES = %w[PARTIALLY_PAID].freeze
+      SUCCESS_STATUSES = %w[PAID].freeze
+      FAILED_STATUSES = %w[EXPIRED CANCELLED].freeze
+
+      def initialize(invoice = nil)
+        @invoice = invoice
+
+        super(nil)
+      end
+
+      def create
+        Rails.logger.debug "invoice.total_amount_cents 0: #{invoice.total_amount_cents}"
+
+        result.invoice = invoice
+        return result unless should_process_payment?
+
+        Rails.logger.debug "invoice.total_amount_cents 1: #{invoice.total_amount_cents.positive?}"
+
+        unless invoice.total_amount_cents.positive?
+          update_invoice_payment_status(payment_status: :succeeded)
+          return result
+        end
+
+        Rails.logger.debug "invoice.total_amount_cents 2: #{invoice.total_amount_cents}"
+
+        increment_payment_attempts
+
+        # res = create_cashfree_payment
```

Comment:
It looks like this file requires a bit of cleanup as well. Could you remove the code that is not in use?